README.md

```diff
@@ -8,23 +8,13 @@ Before getting started, you'll need an account that has CMS Hub Enterprise and a
 
 1. Clone this repo to your machine
 2. Install dependencies via `npm install`
-3. Create a `hubspot.config.yml` file as follows:
-
-```yaml
-defaultPortal: demo
-portals:
-  - name: demo
-    portalId: <portalId>
-    authType: apikey
-    apiKey: <apiKey>
-```
-
+3. If you already have a `hubspot.config.yml` file set up, run `hs auth` to authenticate with your account. Otherwise, run `hs init` to set up your config file.
```

Comment:
I think we should make it clear via a screenshot and other info that they will need to regenerate their access key in order to add custom objects if they already have one.

Reply:
Added a screenshot and more info to this